sonar-batch/src/main/java/org/sonar/batch/issue/IssueFilters.java

```diff
@@ -59,4 +73,145 @@ public boolean accept(DefaultIssue issue) {
       return false;
     }
   }
+
+  private Issue toIssueForIssueFilter(final String componentKey, final BatchReport.Issue rawIssue) {
+    return new Issue() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this anonymous class number of lines from 137 to at most 20, or make it a named class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1188)